ethportal-peertest/src/scenarios/offer_accept.rs

```diff
@@ -45,6 +51,25 @@ pub async fn test_unpopulated_offer(peertest: &Peertest, target: &Client) {
     );
 }
 
+pub async fn test_unpopulated_offer_fails_with_missing_content(
+    peertest: &Peertest,
+    target: &Client,
+) {
+    info!("Testing Unpopulated OFFER/ACCEPT flow with missing content");
+
+    let (content_key, _content_value) = fixture_header_with_proof();
+
+    // validate that unpopulated offer fails if content not available locally
+    assert!(target
+        .offer(
+            Enr::from_str(&peertest.bootnode.enr.to_base64()).unwrap(),
+            content_key.clone(),
+            None,
+        )
+        .await
+        .is_err());
```

Comment:
Would be worth checking the error itself, because it could fail for some other reason, e.g. wrong test configuration, not being connected/able to talk to the target node, etc.